sources/mysql/mysqldump.go

```diff
@@ -49,12 +52,20 @@ var spatialRegexps = func() []*regexp.Regexp {
 var spatialIndexRegex = regexp.MustCompile("(?i)\\sSPATIAL\\s")
 var spatialSridRegex = regexp.MustCompile("(?i)\\sSRID\\s\\d*")
 
+func (ddi DbDumpImpl) GetToDdl() common.ToDdl {
```

Comment:
nit; It's customary to add a comment saying something like "// Functions below implement DbDump interface." just before actual implementation.